atlp-rwanda/falcons-ec-be
# #184637272 Order Status

**What does this PR do?** 
A buyer should be able to track the status of his/her order
Description of Task to be completed?

A buyer should be able to see the status of his/her order 

How should this be manually tested?
clone the repository
Run npm install to install the dependencies and dev dependencies in your terminal
Run npm run dev to start the server in the terminal and check if the server is working
In the URL field enter localhost: PORT/API-docs to open Swagger UI
get order status /api/v1/orders/{order_id} endpoint
**Any background context you want to provide?** 
none 
**What are the relevant pivotal tracker/Trello stories?** 
Tracker: #184637272
**Screenshots (if appropriate)** 
none 
Questions: 
none
